### PR TITLE
e2e: remove e2e related to BTC staking integration

### PR DESCRIPTION
**Pull request by SebastianElvis**
Part of https://github.com/babylonlabs-io/babylon-sdk/issues/64

This PR removes all e2e tests related to BTC staking integration. These tests will then be put in babylon-sdk repository ([PR](https://github.com/babylonlabs-io/babylon-sdk/pull/68))

### CODE REVIEW

**Comment by maurolacy** [comment]
Taking a look at this. What about the e2e tests that depend on babylon? Are you adding babylon container as a dep on the `babylon-sdk` repo?

File: test/e2e/ibc_transfer_e2e_test.go [hunk 1]
```diff
@@ -168,5 +168,5 @@ func (s *IBCTransferTestSuite) Test2IBCTransferBack() {
 		// Check that the balance of the native denom has increased
 		return math.Abs(float64(balanceA.Add(nativeCoin).AmountOf(nativeDenom).Int64()-
 			balanceA2.AmountOf(nativeDenom).Int64())) < delta
-	}, 15*time.Second, 1*time.Second, "Transfer back B was not successful")
+	}, 1*time.Minute, 1*time.Second, "Transfer back B was not successful")
```
Comment (maurolacy) [inline test/e2e/ibc_transfer_e2e_test.go:171]: 👍🏼

**Review by maurolacy — COMMENTED** [review]
👍🏼Less is more.

**Comment by SebastianElvis** [comment]
> Taking a look at this. What about the e2e tests that depend on babylon? Are you adding a babylon container as a dep / tool in the `babylon-sdk` repo?

Yes, there is a [dockerfile](https://github.com/babylonlabs-io/babylon-sdk/pull/68/files#diff-f3a128db95a88e27487ced2f3438d06cd31fbec7796c81fbdb81f7d5cc161769) for babylon node in the Babylon SDK PR.

I'd say that this is less ideal as 1) we might need to refer to a branch rather than a tag for non-tagged changes, and 2) everytime the CI needs to build this image

> Related / follow-up to this: at some point it would be good to change the docker images handling / building model (i.e. the **Docker** section in [babylonlabs-io/pm#118](https://github.com/babylonlabs-io/pm/issues/118)).

Yes, totally agreed on this. Ideally we need a docker registry for all these docker images

**Pull request merged**